<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvToolExtractLung</class>
 <widget class="QWidget" name="vvToolExtractLung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="mLabelInput">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>No selected image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mMaskLoaderBox">
     <property name="title">
      <string>Choose input patient binary mask</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="vvLabelImageLoaderWidget" name="mPatientMaskInputWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vvLabelImageLoaderWidget</class>
   <extends>QWidget</extends>
   <header>vvLabelImageLoaderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../vvIcons.qrc"/>
 </resources>
 <connections/>
</ui>
